### PR TITLE
OSDOCS-11212: Documented the 4.17.0 z-stream RNs

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.17

Issue:
[OSDOCS-11212](https://issues.redhat.com/browse/OSDOCS-11212)

Link to docs preview:
* [4.17.0](https://82638--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-0-ga_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1973,7 +1973,7 @@ With this update, the plugin checks the local cache during the disk-to-mirror pr
 
 * Previously, if a new deployment was done at the OSTree level on the host, which is identical to the current deployment on a different stateroot, OSTree identified them as equal. This behavior prevented the bootloader from updating when the `set-default` command was invoked, because OSTree did not recognize the two stateroots as a differentiating factor for deployments. With this release, OSTree's logic is modified to consider the stateroots. As a result, OSTree properly sets the default deployment to a new deployment that has different stateroots. (link:https://issues.redhat.com/browse/OCPBUGS-30276[*OCPBUGS-30276*]) 
 
-//// 
+////
```
Comment (dfitzmau) [inline release_notes/ocp-4-17-release-notes.adoc:1976]: Had to fix as a large section was incorrectly commented out.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -2747,4 +2747,9 @@ You can view the container images in this release by running the following comma
 ----
 $ oc adm release info 4.17.0 --pullspecs
 ----
+
+[id="ocp-4-17-0-updating_{context}"]
+==== Updating
+To update an {product-title} 4.15 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Comment (cbippley) [inline release_notes/ocp-4-17-release-notes.adoc:2753]:
```suggestion
To update an {product-title} 4.16 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Should probably be 4.16 now.

Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc]: Thank, updated to "4.17".

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -2752,4 +2752,9 @@ You can view the container images in this release by running the following comma
 ----
 $ oc adm release info 4.17.0 --pullspecs
 ----
+
+[id="ocp-4-17-0-updating_{context}"]
+==== Updating
+To update an {product-title} 4.17 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Comment (skopacz1) [inline release_notes/ocp-4-17-release-notes.adoc:2758]: Does this make sense to say 4.17 when this is the first possible version of 4.17? What's the established practice here for .0 z stream versions?

**Review by skopacz1 — COMMENTED** [review]
Just one question, otherwise LGTM

File: release_notes/ocp-4-17-release-notes.adoc (hunk 3 above)
Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc]: I updated it to 4.16 now, so it it more accurate. My apologies.

**Pull request merged**